[OSDOCS-5101]: Replacement control plane nodes in correct group (#56525)
openshift/openshift-docs: release_notes/ocp-4-13-release-notes.adoc

[id="ocp-4-13-cloud-compute-bug-fixes"]
==== Cloud Compute

> * For some configurations of Google Cloud Platform clusters, the internal load balancer uses instance groups that are created by the installation program. Previously, when a control plane machine was replaced manually, the new control plane node was assigned to a compute instance group rather than a control plane instance group. To resolve the issue, administrators had to manually move the control plane machine to the correct instance group by using the Google Cloud console. With this release, replacement control plane nodes are assigned to the correct instance group.
> (link:https://bugzilla.redhat.com/show_bug.cgi?id=1970464[*BZ#1970464*], link:https://issues.redhat.com/browse/OCPCLOUD-1562[*OCPCLOUD-1562*])
> 

Reviewer: ```suggestion
* For some configurations of Google Cloud Platform clusters, the internal load balancer uses instance groups that are created by the installation program. Previously, when a control plane machine was replaced manually, the new control plane node was not assigned to a compute instance group preventing the node being reachable via the internal load balancer. To resolve the issue, administrators had to manually assign the control plane machine to the correct instance group by using the Google Cloud console. With this release, replacement control plane nodes are assigned to the correct instance group.
(link:https://bugzilla.redhat.com/show_bug.cgi?id=1970464[*BZ#1970464*], link:https://issues.redhat.com/browse/OCPCLOUD-1562[*OCPCLOUD-1562*])

```

Author: Thanks for the additional detail on this - let me know what you think of the revision :)
